vector/src/main/java/im/vector/app/features/home/NewHomeDetailFragment.kt

```diff
@@ -206,8 +207,10 @@ class NewHomeDetailFragment :
             newChatBottomSheet.takeIf { !it.isAdded }?.show(requireActivity().supportFragmentManager, NewChatBottomSheet.TAG)
         }
 
-        views.newLayoutOpenSpacesButton.debouncedClicks {
-            spaceListBottomSheet.takeIf { !it.isAdded }?.show(requireActivity().supportFragmentManager, SpaceListBottomSheet.TAG)
+        if (Config.SHOW_SPACES) {
```

Comment:
Would that be interesting to add a comment /* :tchap: */ for each updates we do in the base code?

Reply:
You only disable the click on the button but the purpose of this PR is to **hide** the button.